README.md

```diff
@@ -90,6 +90,7 @@ obtain your session cookie, login to the
 value of the cookie that gets stored in your browser. Put the session number (a
 long hex string) in a file called `.adventofcode.session` in your home
 directory. This file should only contain your session number, in a single line.
+Alternatively you can export your session as the environment variable `ADVENT_OF_CODE_SESSION`.
```

Comment:
It might be worth showing an example of how this can be used to get the token from the keychain or other safe store

Reply:
Well this would require external applications such as: https://github.com/sorah/envchain